widgets/AnaplanFunctionRunner-action.json

```diff
@@ -0,0 +1,89 @@
+{
+  "metadata": {
+    "spec-version": "1.4"
+  },
+  "display-name": "Anaplan Function",
+  "configuration-groups": [
+    {
+      "label": "Function Configuration",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Workspace Id",
+          "name": "workspaceId",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Model Id",
+          "name": "modelId",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "select",
+          "label": "Function Type",
+          "name": "functionType",
+          "widget-attributes": {
+            "width": "large",
+            "default": "PROCESS",
+            "values": [
+              "PROCESS",
+              "ACTION"
+            ]
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Function Name",
+          "name": "functionName",
+          "widget-attributes": {
+            "width": "large"
+          }
+        }
+      ]
+    },
+    {
+      "label": "Anaplan Connection",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Service Location",
+          "name": "serviceLocation",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Auth Service Location",
+          "name": "authServiceLocation",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "User Name",
+          "name": "username",
+          "widget-attributes": {
+            "width": "large"
+          }
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
use the 'password' widget. See https://github.com/cdapio/hydrator-plugins/blob/develop/database-plugins/widgets/Database-batchsource.json#L103 for an example.